pkg/shim/runsc/service.go

```diff
@@ -49,7 +51,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sys/unix"
 	"gvisor.dev/gvisor/pkg/cleanup"
-	"gvisor.dev/gvisor/pkg/shim/runtimeoptions/v14"
+	v14 "gvisor.dev/gvisor/pkg/shim/runtimeoptions/v14"
```

Comment:
it is not needed

Reply:
@Champ-Goblem do you have a chance to update the PR?

I hope to rebase https://github.com/google/gvisor/pull/11468 on top of the PR.

Reply:
I have just force-pushed the PR now

Reply:
thanks a lot!